existing-communities/README.md

```diff
@@ -1 +1,47 @@
+
+**Practical Communities**  
+Practical communities are special because they don't center around ideals at all. Practical communities arise out of pure convenience, when people are better of sharing information or resources (on a regulare basis). This can be a community of farmers sharing farming equipment, or students sharing a house.
+
+### MMO-RPG Communities
+Massive Multiplayer Online Role-Playing Games are, like the name describes, online games where the player explores a virtual world as one of many type of characters and can interact with an enormous amount of other players. Popular MMO-RPG's often have two types of communites: communities outside of the game (fans and forums) and communities inside of the game (parties and guilds). For this research, we will be looking at the communities inside the game, because the dynamics of these communities often require active participation.  
+Most MMO-RPG's have two in-game systems that can be seen as communities: parties and guilds. A party is a small group of players that train or do quests (in-game missions) together for a short period of time (possibly one game session). The benefit of forming a party is experience-sharing (to some degree) and players can kill high-level in-game creatures they could kill on their own. If a party is formed for the purpose of doing a quest, the composition of the party (the distribution of roles or functions) may be addapted towards the goal of the quest to ensure optimal performance{{"chen2009communication" | cite}}.  
+A guild is a group of players forming a community, for creating a social connection or to make it easier to form a trust-worthy or well-composed party over a longer period of time (longer than one game session){{"rossi2008mmorpg" | cite}}{{"kelly2004play" | cite}}{{"verhagen2009demystifying" | cite}}. Within a guild, members can be classified based on connectivity, type of player, role within the guild and involvement{{"ang2010social" | cite}}.
+
+### Software Communities
+Software communities are groups of people involved in the development or improvement of a piece of software, but can also be focussed on a programing language (model). This makes software communities a broad term. For this research, we will be looking at Software Development Communities,  communities centered around software development/improvement, and we won't be looking at Software Engineering Communities,  communities centered around programing language (models), because the first community is our target group.   
+
+A software development communities can be devided into two groups: developers and users. Developers create software, wich is used by the users. The users are able to give feedback about the software, wich the developers can use in the next version of that piece of software.
```

Comment:
`A software development communities can be devided into two groups` ==> `Software development communities can be divided into two groups`